README.md

```diff
@@ -22,6 +26,12 @@ Make sure your project meets the minimum requirements:
 
 `flutter pub add viam_sdk`
 
+## Upgrading
+
+`flutter pub upgrade viam_sdk`
+
+Note that breaking changes may occur in patch version changes, so we recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.
```

Comment:
I would mention that breaking changes may occur in patch version changes BECAUSE we're still in beta.

Reply:
```suggestion
Note that the TypeScript SDK is still in beta, so breaking changes may occur in patch version changes.
We recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.
```